src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsTrackingOptInRequestsOasis.java

```diff
@@ -20,311 +20,311 @@
-  }
+//  @NetworkTrafficDump(useMITM = true)
+//  @Execute(trackingOptIn = false)
+//  @Test(
+//      dataProviderClass = TrackingOptInDataProvider.class,
+//      dataProvider = "adsKikimoraRejectedDataProvider",
+//      groups = "AdsOptInRejectedOasis"
+//  )
+//  public void adsTrackingRejectedForKikimora(String[] instantGlobals, List<String> urlPatterns) {
+//    networkTrafficInterceptor.startIntercepting();
+//    TrackingOptInModal modal = new TrackingOptInModal();
+//    modal.rejectOptInModal(driver, POLAND, ADS_HOME_PAGE, instantGlobals);
+//
+//    modal.verifyTrackingRequestsSend(urlPatterns, networkTrafficInterceptor);
+//  }
+//
+//  @NetworkTrafficDump(useMITM = true)
+//  @Execute(trackingOptIn = false)
+//  @Test(
+//      dataProviderClass = TrackingOptInDataProvider.class,
+//      dataProvider = "adsKruxDataProvider",
+//      groups = "AdsOptInRejectedOasis"
+//  )
+//  public void adsTrackingRejectedForKrux(String[] instantGlobals, List<String> urlPatterns) {
+//    networkTrafficInterceptor.startIntercepting();
+//    TrackingOptInModal modal = new TrackingOptInModal();
+//    modal.rejectOptInModal(driver, POLAND, ADS_HOME_PAGE, instantGlobals);
+//
+//    modal.verifyTrackingRequestsNotSend(urlPatterns, networkTrafficInterceptor);
+//  }
+//
+//  @NetworkTrafficDump(useMITM = true)
+//  @Execute(trackingOptIn = false)
+//  @Test(
+//      dataProviderClass = TrackingOptInDataProvider.class,
+//      dataProvider = "adsNetzAthletenDataProvider",
+//      groups = "AdsOptInRejectedOasis"
+//  )
+//  public void adsTrackingRejectedForNetzAthleten(String[] instantGlobals,
+//                                                 List<String> urlPatterns) {
+//    networkTrafficInterceptor.startIntercepting();
+//    TrackingOptInModal modal = new TrackingOptInModal();
+//    modal.rejectOptInModal(driver, GERMANY, ADS_HOME_PAGE, instantGlobals);
+//
+//    modal.verifyTrackingRequestsNotSend(urlPatterns, networkTrafficInterceptor);
+//  }
+//
+//  @NetworkTrafficDump(useMITM = true)
+//  @Execute(trackingOptIn = false)
+//  @Test(
+//      dataProviderClass = TrackingOptInDataProvider.class,
+//      dataProvider = "adsMoatDataProvider",
+//      groups = "AdsOptInRejectedOasis"
+//  )
+//  public void adsTrackingRejectedForMoat(String[] instantGlobals, List<String> urlPatterns) {
+//    networkTrafficInterceptor.startIntercepting();
+//    TrackingOptInModal modal = new TrackingOptInModal();
+//    modal.rejectOptInModal(driver, POLAND, ADS_HIVI_PAGE, instantGlobals);
+//
+//    modal.verifyTrackingRequestsSend(urlPatterns, networkTrafficInterceptor);
+//  }
+//
+//  @NetworkTrafficDump(useMITM = true)
+//  @Execute(trackingOptIn = false)
+//  @Test(
+//      dataProviderClass = TrackingOptInDataProvider.class,
+//      dataProvider = "adsNordicsDataProvider",
+//      groups = "AdsOptInRejectedOasis"
+//  )
+//  public void adsTrackingRejectedForNordics(List<String> urlPatterns) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ACommentedOutCodeLine)